### PR TITLE
Docs warn navigator ios

### PR DESCRIPTION
**Pull request by morenoh149**
ping https://github.com/facebook/react-native/issues/795#issuecomment-139895307

### CODE REVIEW

**Comment by brentvatne** [comment]
lgtm cc @foghina

**Comment by foghina** [comment]
cc @sahrens

**Comment by brentvatne** [comment]
Hi there! Please rebase this and I will merge :)

File: Libraries/Components/Navigation/NavigatorIOS.ios.js [hunk 1]
```diff
@@ -95,6 +95,13 @@ type Event = Object;
  * NavigatorIOS wraps UIKit navigation and allows you to add back-swipe
  * functionality across your app.
  *
+ * > **NOTE**: This Component is not maintained by Facebook
+ * >
+ * > This component is under community responsibility and currently suffers from
+ * > a rendering [bug](https://github.com/facebook/react-native/issues/795).
+ * > If a pure javascript solution fits your needs you may try the `Navigator`
```
Comment (ide) [inline Libraries/Components/Navigation/NavigatorIOS.ios.js]: JavaScript

File: Libraries/Components/Navigation/NavigatorIOS.ios.js [hunk 2]
```diff
@@ -95,6 +95,13 @@ type Event = Object;
  * NavigatorIOS wraps UIKit navigation and allows you to add back-swipe
  * functionality across your app.
  *
+ * > **NOTE**: This Component is not maintained by Facebook
+ * >
+ * > This component is under community responsibility and currently suffers from
+ * > a rendering [bug](https://github.com/facebook/react-native/issues/795).
```
Comment (ide) [inline Libraries/Components/Navigation/NavigatorIOS.ios.js]: Leave off the bug.. there are several bugs with NavigatorIOS

**Comment by mkonicek** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**